kinto_http/__init__.py

```diff
@@ -259,6 +267,89 @@ def delete_buckets(self, safe=True, if_match=None, if_exists=False):
+                                              group=group,
+                                              bucket=bucket,
+                                              data=data,
+                                              permissions=permissions,
+                                              safe=safe)
+        headers = DO_NOT_OVERWRITE if safe else None
+        endpoint = self.get_endpoint('group',
+                                     bucket=bucket,
+                                     group=group)
+        try:
+            resp, _ = self.session.request('put', endpoint, data=data,
+                                           permissions=permissions,
+                                           headers=headers)
+        except KintoException as e:
+            if e.response.status_code == 403:
+                msg = ("Unauthorized. Please check that the bucket exists and "
+                       "that you have the permission to create or write on "
+                       "this group.")
+                e = KintoException(msg, e)
+            raise e
+
+        return resp
+
+    def update_group(self, data=None, group=None, bucket=None,
+                     permissions=None, method='put',
+                     safe=True, if_match=None):
+        endpoint = self.get_endpoint('group',
+                                     bucket=bucket,
+                                     group=group)
+        headers = self._get_cache_headers(safe, data, if_match)
+        resp, _ = self.session.request(method, endpoint, data=data,
+                                       permissions=permissions,
+                                       headers=headers)
+        return resp
+
+    def patch_group(self, *args, **kwargs):
+        kwargs['method'] = 'patch'
+        return self.update_group(*args, **kwargs)
+
+    def get_group(self, group=None, bucket=None):
+        endpoint = self.get_endpoint('group',
+                                     bucket=bucket,
+                                     group=group)
+        resp, _ = self.session.request('get', endpoint)
+        return resp
+
+    def delete_group(self, group=None, bucket=None,
+                     safe=True, if_match=None, if_exists=False):
+        if if_exists:
+            return self._delete_if_exists('group',
+                                          group=group,
+                                          bucket=bucket,
+                                          safe=safe,
+                                          if_match=if_match)
+        endpoint = self.get_endpoint('group',
+                                     bucket=bucket,
+                                     group=group)
+        headers = self._get_cache_headers(safe, if_match=if_match)
+        resp, _ = self.session.request('delete', endpoint, headers=headers)
+        return resp['data']
+
+    def delete_groups(self, bucket=None, safe=True, if_match=None, if_exists=False):
+        if if_exists:
```

Comment:
Just omit it in `delete_groups()` for now..
Usually when we find an inconsistency during a session of coding we try to restrain ourselves to fix it right away, so that the changes of the current pull-request remains atomic and focused on one thing. 

Do not hesitate to open as many issues as you want with everything you spot in the code, we will fix them afterwards once this one is merged! (for example look at one I had open a while ago in another repo :) https://github.com/Kinto/kinto-admin/issues/204)